system/handlers/formcontrols/FileTypePicker.cfc

```diff
@@ -16,7 +16,7 @@ component {
 			for( var type in assetTypes[ assetTypeGroup ] ){
 				types.append( {
 					  value = type
-					, label = translateResource( "filetypes:#type#.picker.label" )
+					, label = translateResource( "filetypes:#type#.picker.label", translateResource( "filetypes:#assetTypeGroup#.picker.text", "Unknown" ) & ": #type#" )
```

Comment:
If we are going to do defaults here, we need to default everything. So two extra i18n URIs: 

```properties
unknown.group.type=Unknown
default.type.label={1}: {2}
```

Then, here:

```cfc
groupType = translateResource( "filetypes:#assetTypeGroup#.picker.text", translateResource( "filetypes:unknown.group.type" ) );
default = translateResource( uri="default.type.label", data=[ groupType, type ]  );
// ...
label = translateResource( "filetypes:#type#.picker.label", default )
```

Reply:
Update with additional suggested i18n and related changes.